rust/build.rs

```diff
@@ -0,0 +1,10 @@
+use gdnative_project_utils::*;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let classes = scan_crate("src")?;
+    Generator::new()
+        .godot_project_dir("../godot/native")
```

Comment:
This makes `project-utils` generate into `godot/native/native`, instead of `godot/native`.

Reply:
done...